packages/vuetifyjs.com/src/pages/framework/InternationalizationPage.vue

```diff
@@ -30,8 +36,8 @@
         |
         | Vue.use(Vuetify, {
         |   lang: {
-        |     locales: { 'zh-Hans': zhHans, pl, sv },
-        |     current: 'zh-Hans'
+        |     locales: { zhHans, pl, sv },
+        |     current: 'zhHans'
```

Comment:
See https://github.com/vuetifyjs/vuetifyjs.com/pull/644, imports should be updated too (es5 -> lib, member import)